accumulo-store/src/test/java/gaffer/accumulostore/key/core/impl/AbstractAccumuloElementConverterTest.java

```diff
@@ -93,24 +108,34 @@ public void TestAccumuloKeyConverterCFCQPropertydEdge() throws SchemaException,
 
     @Test
     public void TestAccumuloKeyConverterCFCQPropertydEntity() throws SchemaException, AccumuloElementConversionException, IOException {
```

Comment:
Method names should start with a lower case letter. If possible test methods should use the format 'shouldReturnXWhenY'.